File: auto-docs/executables/julia/basic_histogram.jl
# Learn about API authentication here: plot.ly/julia/getting-started
# Find your api_key here: plot.ly/settings/api

using Plotly

x = randn(500)

Plotly.signin("TestBot", "r1neazxo9w")
data = [
  [
    "x" => x, 
    "type" => "histogram"
  ]
]
response = Plotly.plot(data, ["filename" => "basic-histogram", "fileopt" => "overwrite"])
plot_url = response["url"]

File: auto-docs/executables/matlab/basic_histogram.m
% Learn about API authentication here: plot.ly/matlab/getting-started
% Find your api_key here: plot.ly/settings/api

signin('TestBot', 'r1neazxo9w')

x = randn(500,1);

data = {...
  struct(...
    'x', x, ...
    'type', 'histogram')...
};
response = plotly(data, struct('filename', 'basic-histogram', 'fileopt', 'overwrite'));
plot_url = response.url
